### DISCUSSION TITLE
Warning in XRechnung UBL

### DISCUSSION THREAD

**Discussion by CSharpProgrammerA** [stephanstapel/ZUGFeRD-csharp#396] [General]
Hello,

I set the value “BusinessProcess” to 

desc.BusinessProcess = “urn:fdc:peppol.eu:2017:poacc:billing:01:1.0”;

Nevertheless, I get the warning :
[BR-DE-21] Das Element "Specification identifier" (BT-24) soll syntaktisch der Kennung des Standards XRechnung entsprechen.

the warning does not appear with CII .

What can you do about it?

Thanks Andre

**Comment by CSharpProgrammerA** [comment]
Hello,

in the file :
InvoiceDescriptor22UBLWriter.cs

I changed this line 
Writer.WriteElementString("cbc:CustomizationID", "urn:cen.eu:en16931:2017#compliant#urn:xeinkauf.de:kosit:xrechnung_3.0");

before it was this:
Writer.WriteElementString("cbc:CustomizationID", "urn:cen.eu:en16931:2017#compliant#urn:xoev-de:kosit:standard:xrechnung_3.0");

Now I do not get a warning !!!

regards Andreas

**Comment by stephanstapel** [comment]
agreed. @CSharpProgrammerA, can you please create a PR?
